Source/Patches/Core/StartOfRoundPatch.cs

```diff
@@ -21,5 +22,13 @@ public static void PositionSuitsOnRackPostfix(ref StartOfRound __instance)
                 suitHanger.AddComponent<PlayerModelSwitcher>();
             }
         }
+
+        [HarmonyPatch("OnClientDisconnect")]
+        [HarmonyPrefix]
+        public static void OnClientDisconnectPostfix(ref ulong clientId)
+        {
+            // Reset the status of this player in case they reconnect
+            clientId.GetPlayerController().GetComponent<PlayerModelReplacer>().EnableHumanModel(false);
```

Comment:
This patch fixes a connectivity bug when people disconnect while in dog mode and then reconnect.